lang/cs/Org.Apache.REEF.Client/YARN/YarnCommandLineEnvironment.cs

```diff
@@ -87,7 +96,7 @@ internal string GetYarnCommandPath()
         /// </summary>
         /// <param name="arguments"></param>
         /// <returns>Whatever was printed to stdout by YARN.</returns>
-        internal string Yarn(params string[] arguments)
+        private string Yarn(params string[] arguments)
```

Comment:
Yes. I propose we leave this as is for now, using `YarnCommandLineEnvironment`. For HDInsight submission we can add new implementation of `IYarnCommandLineEnvironment`?

I want us to reach the point of E2E testing of client with YARN REST API and job resource uploader (in the environments it works currently) as soon as we can as first checkpoint. And then get HDInsight client working.